nemo_aligner/algorithms/critic_server_trainer.py

```diff
@@ -72,11 +72,12 @@ def __init__(self, cfg, model, optimizer, scheduler, logger, ckpt_callback, gbs)
             Tensor(name="sequence_lengths", shape=(-1,), dtype=np.int64, optional=True),
             Tensor(name="add_EOS", shape=(1,), dtype=np.bool_, optional=True),
         )
-        self.infer_outputs = (
+        self.infer_outputs = [
             Tensor(name="values", shape=(-1,), dtype=np.float32),
```

Comment:
nit: can we do the same logic but with "values" key instead?

it feels more intuitive to add on the "value" key rather than the "reward" key when  `combine_rm_and_critic_server=True`

Reply:
Sorry, I'm not sure I follow -- values are always returned by the critic, what do you mean exactly?

Reply:
ah! my appologies, I don't know what I was thinking -- this looks good to me